webextensions/manifest.json

```diff
@@ -19,30 +19,30 @@
     "bookmarks"
   ],
   "icons": {
-    "16": "resources/16x16-light.svg",
-    "20": "resources/20x20-light.svg",
-    "24": "resources/24x24-light.svg",
-    "64": "resources/icon64.png",
-    "94": "resources/icon94.png"
+    "16": "resources/icon.svg",
+    "20": "resources/icon.svg",
+    "24": "resources/icon.svg",
+    "64": "resources/icon",
+    "94": "resources/icon"
   },
   "background": {
     "page": "background/background.html"
   },
   "browser_action": {
     "default_title": "__MSG_sidebarTitle__",
     "default_icon": {
```

Comment:
According to https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/browser_action#Choosing_icon_sizes 
> You can supply a single icon as an SVG file, and it will be scaled correctly

so this section could just be `"default_icon": "resources/icon.svg",`